extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -87,6 +104,31 @@ public void build(ApplicationInfoBuildItem applicationInfo,
         featureProducer.produce(new FeatureBuildItem(FeatureBuildItem.KUBERNETES));
     }
 
+    private void dekorate(Session session, ApplicationInfoBuildItem applicationInfo,
+            KubernetesConfig kubernetesConfig,
```

Comment:
It seems like this is no longer being used, that doesn't look right...

Reply:
I still don't see this being used which unless I am mistaken, that the quarkus configuration will be ignored entirely.